lib/parsers/asm-parser-spirv.ts

```diff
@@ -105,13 +105,22 @@ export class SPIRVAsmParser extends AsmParser {
         const opLine = /OpLine/;
```

Comment:
It's such a shame we don't have tests for this :( (not your fault, just commenting). I filed https://github.com/compiler-explorer/compiler-explorer/issues/7177 to track.

Reply:
I actually thought the same thing and realized there was a test framework for it, it was on one of my next TODOs to get some in for all the SPIR-V producing items